### DISCUSSION TITLE
ImportError: DLL load failed while importing _dlib_pybind11: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by jaisak27** [davisking/dlib#2898] [Q&A]
when trying to use dlib this message is showing

```
 import dlib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\NEERAJ\AppData\Local\Programs\Python\Python311\Lib\site-packages\dlib\__init__.py", line 19, in <module>
    from _dlib_pybind11 import *
ImportError: DLL load failed while importing _dlib_pybind11: The specified module could not be found.
```

the file _dlib_pybind11 is present but still not able to use

**Comment by davisking** [comment]
Try reinstalling dlib.  That error indicates it isn't installed properly.  Make sure you let the official dlib installer do it (e.g. run pip install dlib) rather than following any other 3rd party instructions. Especially ones that talk about moving files around manually.

**Comment by jaisak27** [comment]
I have tried to reinstall dlib multiple times, but the problem remains,
what to do now?
